templates/page.html

```diff
@@ -182,22 +182,17 @@ <h3>Select a REANA cluster...</h3>
         <h3>...or install your own</h3>
         <figure class="figure">
           <div class="command-line">
-            <div class="line comment"># install kubectl 1.16.3, minikube 1.5.2 and helm 3.0.0</div>
-            <div class="line"><span class="prompt">$</span>sudo dpkg -i kubectl*.deb minikube*.deb kubernetes-helm*.deb</div>
-            <div class="line comment"># start minikube</div>
-            <div class="line"><span class="prompt">$</span>minikube start --feature-gates="TTLAfterFinished=true"</div>
-            <div class="line comment"># create new virtual environment</div>
-            <div class="line"><span class="prompt">$</span>virtualenv ~/.virtualenvs/myreana</div>
-            <div class="line"><span class="prompt">$</span>source ~/.virtualenvs/myreana/bin/activate</div>
-            <div class="line comment"># install reana-cluster utility</div>
-            <div class="line"><span class="prompt">$</span>pip install reana-cluster</div>
-            <div class="line comment"># deploy new cluster and check progress</div>
-            <div class="line"><span class="prompt">$</span>reana-cluster init</div>
-            <div class="line"><span class="prompt">$</span>reana-cluster status</div>
-            <div class="line comment"># set environment variables for client</div>
-            <div class="line"><span class="prompt">$</span>eval $(reana-cluster env --include-admin-token)</div>
+            <div class="line comment"># install kubectl 1.16+, kind 0.8+ and helm 3.0+</div>
+            <div class="line"><span class="prompt">$</span>sudo dpkg -i kubectl*.deb kind*.deb kubernetes-helm*.deb</div>
+            <div class="line comment"># create Kubernetes cluster (or use your own!)</div>
+            <div class="line"><span class="prompt">$</span>wget https://raw.githubusercontent.com/reanahub/reana/0.7.0/etc/kind-example.yaml</div>
+            <div class="line"><span class="prompt">$</span>kind create cluster --config kind-cluster-localhost.yaml</div>
```

Comment:
I think you forgot to update the filename here.
```suggestion
            <div class="line"><span class="prompt">$</span>kind create cluster --config kind-example.yaml</div>
```